src/main/java/com/exasol/adapter/dynamodb/DynamodbAdapter.java

```diff
@@ -0,0 +1,166 @@
+package com.exasol.adapter.dynamodb;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Logger;
+
+import com.exasol.ExaConnectionAccessException;
+import com.exasol.ExaConnectionInformation;
+import com.exasol.ExaMetadata;
+import com.exasol.adapter.AdapterException;
+import com.exasol.adapter.AdapterProperties;
+import com.exasol.adapter.VirtualSchemaAdapter;
+import com.exasol.adapter.capabilities.Capabilities;
+import com.exasol.adapter.metadata.ColumnMetadata;
+import com.exasol.adapter.metadata.DataType;
+import com.exasol.adapter.metadata.SchemaMetadata;
+import com.exasol.adapter.metadata.TableMetadata;
+import com.exasol.adapter.request.*;
+import com.exasol.adapter.response.*;
+
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClientBuilder;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.ScanRequest;
+import software.amazon.awssdk.services.dynamodb.model.ScanResponse;
+
+/**
+ * DynamoDB Virtual Schema adapter
+ */
+public class DynamodbAdapter implements VirtualSchemaAdapter {
+
+	private static final Logger LOGGER = Logger.getLogger(DynamodbAdapter.class.getName());
+
+	/**
+	 * creates a hard coded table "testTable" with only one column testCol
```

Comment:
Use Capitalization and punctuation consistently in comments. Wrap identifiers and literals in `{@code }`. Please repeat for all Javadoc comments.